WalletWasabi/Tor/Socks5/TorSocks5Client.cs

```diff
@@ -171,7 +171,7 @@ private async Task HandshakeAsync(string identity)
 			Logger.LogDebug("<");
 		}
 
-		internal async Task ConnectToDestinationAsync(EndPoint destination)
+		public async Task ConnectToDestinationAsync(EndPoint destination)
```

Comment:
Shouldn't this one be private?

Reply:
You are right. That can be private too. 👍